src/media/shortcodes/class-wpml-page-builders-media-shortcodes.php

```diff
@@ -3,8 +3,8 @@
 class WPML_Page_Builders_Media_Shortcodes {
 
 	const ALL_TAGS = '\w+';
```

Comment:
@strategio I don't think is used anymore.

Reply:
Done @brucepearson (see https://github.com/OnTheGoSystems/wpml-page-builders/pull/33/commits/464fa49bf120f96cbbcc4bd346ff02a9bd9ac0d7). I removed the `*` wildcard  and the `|` which were giving the ability to match several tag names.